src/generic-provider-views/AuthView.js

```diff
@@ -7,7 +7,7 @@ module.exports = (props) => {
       <h1 class="UppyProvider-authTitle">
         Please authenticate with <span class="UppyProvider-authTitleName">${props.pluginName}</span><br> to select files
       </h1>
-      <a class="UppyProvider-authBtn" href=${props.link} target="_blank">Authenticate</a>
+      <a class="UppyProvider-authBtn" onclick=${props.handleAuth}>Authenticate</a>
```

Comment:
🤔 Since this is not a link now, semantically we could turn it into a `<button>`, making sure to include `@include reset-button` in its style declaration in SCSS, which will reset all sorts of browser default button styles.